editor.planx.uk/src/@planx/components/MapAndLabel/Public/CopyFeature.tsx

```diff
@@ -43,16 +43,16 @@ export const CopyFeature: React.FC<Props> = ({
       onSubmit={formik.handleSubmit}
     >
       <Box>
-        <InputLabel label="Copy from" id={`select-${destinationIndex}`}>
+        <InputLabel label="Copy from" id={`select-${destinationIndex}-label`}>
           <SelectInput
             disabled={isDisabled}
+            name={`select-${destinationIndex}`}
```

Comment:
This is the fix for the main issue flagged in the report -
 - Redundant hidden label removed from `<SelectInput/>`
 - Ids correct associated (`SelectInput` appends "-label" to `aria-labelledby)

<img width="286" height="172" alt="Image" src="https://github.com/user-attachments/assets/f7675c5b-4fca-4c66-86d7-c14c6d1b0b03" />